modular_zubbers/code/datums/weather/weather_types/radiation_storm.dm

```diff
@@ -9,3 +9,13 @@
 		/area/ruin/unpowered/primitive_catgirl_den,
 	))
 	. = ..()
+
+
+/datum/weather/rad_storm/New()
+	. = ..()
+
+	for (var/area/station/commons/dorms/dorm_subtype in subtypesof(/area/station/commons/dorms))
+		protected_areas += dorm_subtype
+
+	protected_areas += /area/ruin/space/has_grav/bubbers/dauntless
+	protected_areas += /area/ruin/space/has_grav/bubbers/dauntless_space
```

Comment:
Pile these into the lazyor above, also it already includes subtypes so no need for the loop

Reply:
commons/dorms protects ALL of dorms, im aiming to protect only rooms

if youre fine with protecting all of dorms, i can make it simpler but yeah

Reply:
I feel like just having all of dorms protected